### DISCUSSION TITLE
How could I have a file field available for contacts?

### DISCUSSION THREAD

**Discussion by OscarGS86** [YetiForceCompany/YetiForceCRM#15286] [How to] [answered]
Hi guys,

I would like to know how I can have a file field available for the contacts. I would like to attach a PDF or WORD file to each of my contacts. 

I have been checking the fields available and It seems that there is not any field to allow me to attach a file against each contact profile.

Could anyone help me with this question?

Thank you so much in advance,

**Comment by mariuszkrzaczkowski** [comment]
there is no such field, it is only for graphic files, you can attach with documents

you can expand the current field for photos and do so that you can attach any format, probably a few hours of work and it will do the trick

**Reply by OscarGS86** [reply]
Thanks, @mariuszkrzaczkowski ,

Is there any guide where I can find where I can make the changes to make that field available for PDF files?

I am a newbie using this software (YetiForce CRM).

Cheers,

**Reply by mariuszkrzaczkowski** [reply]
select a field from the variable panel

**Comment by rskrzypczak** [comment, marked as answer]
> I would like to attach a PDF or WORD file to each of my contacts.

The only possibility is through the Documents module
![image](https://user-images.githubusercontent.com/10221999/136736557-e06ef4a2-0651-442f-93b0-e7217c4d904c.png)
